doc/user_guide/user_guide.md

```diff
@@ -82,23 +82,59 @@ cat language_container_part_* > language_container.tar.gz
 ```
 
 #### Install Language Container
-- To install the language container, it is necessary to load the container into the BucketFS 
-and register it to the database. The following command provides this setup:
-```buildoutcfg
-python -m exasol_transformers_extension.deploy language-container
-    --dsn <DB_HOST:DB_PORT> \
-    --db-user <DB_USER> \
-    --db-pass <DB_PASSWORD> \
-    --bucketfs-name <BUCKETFS_NAME> \
-    --bucketfs-host <BUCKETFS_HOST> \
-    --bucketfs-port <BUCKETFS_PORT> \
-    --bucketfs-user <BUCKETFS_USER> \
-    --bucketfs-password <BUCKETFS_PASSWORD> \
-    --bucket <BUCKETFS_NAME> \
-    --path-in-bucket <PATH_IN_BUCKET> \
-    --language-alias <LANGUAGE_ALIAS> \ 
-    --container-file <path/to/language_container.tar.gz>       
-```
+You can install the language container in two different methods, described below:
+
+  1. *Installation with Python Script*
+
+     To install the language container, it is necessary to load the container 
+     into the BucketFS and register it to the database. The following command 
+     provides this setup using the python script provided with this library:
+
+      ```buildoutcfg
+      python -m exasol_transformers_extension.deploy language-container
+          --dsn <DB_HOST:DB_PORT> \
+          --db-user <DB_USER> \
+          --db-pass <DB_PASSWORD> \
+          --bucketfs-name <BUCKETFS_NAME> \
+          --bucketfs-host <BUCKETFS_HOST> \
+          --bucketfs-port <BUCKETFS_PORT> \
+          --bucketfs-user <BUCKETFS_USER> \
+          --bucketfs-password <BUCKETFS_PASSWORD> \
+          --bucket <BUCKETFS_NAME> \
+          --path-in-bucket <PATH_IN_BUCKET> \
+          --language-alias <LANGUAGE_ALIAS> \ 
+          --container-file <path/to/language_container.tar.gz>       
+      ```
+
+  2. *Manual Installation*
+
+     In the manual installation, the pre-built container should be firstly 
+     uploaded into BucketFS. In order to do that, you can use 
+     either a [http(s) client](https://docs.exasol.com/database_concepts/bucketfs/file_access.htm) 
+     or the [bucketfs-client](https://github.com/exasol/bucketfs-client). 
+     The following command uploads a given container into BucketFS through curl 
+     command, a http(s) client: 
+      ```buildoutcfg
+      curl -vX PUT -T \ 
+          "<CONTAINER_FILE>" 
+          "http://w:<BUCKETFS_WRITE_PASS>@$bucketfs_host:<BUCKETFS_PASS>/<BUCKETFS_NAME>/<PATH_IN_BUCKET><CONTAINER_FILE>"
+      ```
+
```

Comment:
````suggestion

Please note that specifying the password on command line will make your shell record the password in the history. To avoid leaking your password please consider to set an environment variable. The following examples sets environment variable `bucketfs_write_password`:
```shell 
read -sp "password: " bucketfs_write_password
```
````